CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+# Default
+*				@gdbelvin
+
+/cmd				@gdbelvin
+/core				@gdbelvin
+/core/crypto			@gdbelvin
+/deploy				@gdbelvin
+/docs				@gdbelvin
+/impl				@gdbelvin
```

Comment:
Why not let the default case cover these?

Reply:
reminding myself of the right syntax, but that isn't a very good reason. Removed.